pvlib/pvsystem.py

```diff
@@ -2909,22 +2900,105 @@ def pvwatts_dc(effective_irradiance, temp_cell, pdc0, gamma_pdc, temp_ref=25.):
     temp_ref: numeric, default 25.0
         Cell reference temperature. PVWatts defines it to be 25 C and
         is included here for flexibility. [C]
```

Comment:
Idk about you Will, but I prefer my temps in degrees celsius over coulombs ;P
```suggestion
        Cell reference temperature. PVWatts defines it to be 25 °C and
        is included here for flexibility. [°C]
```